Update Ansible to the latest version and Python 3.12 (#5)
ome/ansible-role-python3-virtualenv: README.md

In some situations, particularly if `ansible_python_interpreter` is set, the Ansible pip modules passes unrecognised parameters.
This role installs a wrapper script `/usr/local/bin/ome-python3-virtualenv` should work in all cases.

Role Variables
--------------
python_version: The default Python version value is 3.12, with support also available for Python 3.11.

Example Playbooks
-----------------
    
    - hosts: localhost
      roles:
      - role: ome.ansible-role-python3-virtualenv
        # Uncomment the line below to use Python 3.11. 

>         # python_version: "3:11"

Reviewer: ``3:11`` -> ``3.11`` 

Possible typo @khaledk2  ^^^ ?

Author: Fixed on [9ea084a](https://github.com/ome/ansible-role-python3-virtualenv/pull/5/commits/9ea084aac0b1730bcd6e012d1244fb95f8b34290)

In some situations, particularly if `ansible_python_interpreter` is set, the Ansible pip modules passes unrecognised parameters.
This role installs a wrapper script `/usr/local/bin/ome-python3-virtualenv` should work in all cases.

The role first creates a new virtual environment, then creates or updates a symlink so that the specified `env_name` points to `venv-python_version`.

> python_version: The default Python version value is 3.12, with support also available for Python 3.11.

Reviewer: Please add inverted commas around each variable to fix formatting

> virtualenv_basedir: Base directory where the virtual environment is located

Reviewer: inverted commas please around `virtualenv_basedir`

> env_name: The virtual environment name, the default value is venv3

Reviewer: inverted commas please around `env_name`

In some situations, particularly if `ansible_python_interpreter` is set, the Ansible pip modules passes unrecognised parameters.
This role installs a wrapper script `/usr/local/bin/ome-python3-virtualenv` should work in all cases.

> The role first creates a new virtual environment, then creates or updates a symlink so that the specified env_name points to venv-python_version.

Reviewer: ```suggestion
The role first creates a new virtual environment, then creates or updates a symlink so that the specified `env_name` points to `venv-python_version`.
```

Reviewer: just format the vars as code ^^^

> Let me know if you'd like a shorter or more technical phrasing.

Reviewer: this is fine imho, thank you

Reviewer: ```suggestion

```

Author: Removed

> \q--------------

Reviewer: ```suggestion

```

Author: Removed

Role Variables
--------------
python_version: The default Python version value is 3.12, with support also available for Python 3.11.
virtualenv_basedir: Base directory where the virtual environment is located
service_name: service name, e.g. omero-server
env_name: The virtual environment name, the default value is venv3

Example Playbooks 
-----------------

>     # Example for use during OMERO.server installation or upgrade

Reviewer 2: ```suggestion
    # Example to use during an OMERO.server installation or upgrade
```

    - hosts: localhost
      roles:
      - role: ome.ansible-role-python3-virtualenv
        # Uncomment the line below to use Python 3.11. 
        # python_version: "3.11"
        virtualenv_basedir: /opt/omero/server
        service_name: omero-server

>     #  Example for use during OMERO.web installation or upgrade        

Reviewer 2: ```suggestion
    #  Example to use during an OMERO.web installation or upgrade        
```
